Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py

```diff
@@ -365,85 +360,34 @@ def comments_to_notes(client: Client, comments: List):
     Turns comments into XSOAR entries
     """
 
-    def to_note(comment, filenames):
+    def to_note(comment):
         occurred = comment["datetime_created"]
         oid = comment["id"]
-        readable = {
-            "occurred": occurred,
-            "contents": comment["comment"],
-            "type": "comment",
-            "files": filenames,
-        }
+        user_display = user_to_display(comment["user"])
+        footer = f"\n\nSent by {user_display} via ZTAP"
         return {
             "Type": EntryType.NOTE,
             "ContentsFormat": EntryFormat.JSON,
             "Contents": comment,
-            "HumanReadable": readable,
-            "ReadableContentsFormat": EntryFormat.JSON,
+            "HumanReadable": comment["comment"] + footer,
+            "ReadableContentsFormat": EntryFormat.TEXT,
```

Comment:
Why did you change it from a json to text?

Reply:
Originally, we included both the audit logs and comments as notes, however the audit logs ended up cluttering incidents and made reading the actual comments more difficult. Moving the comments to plaintext allows reading the comment directly instead of having to expand the json object.